relaxed.module

```diff
@@ -65,7 +65,7 @@ function relaxed_entity_access(EntityInterface $entity, $operation, AccountInter
   $entity_type_id = $entity_type->id();
 
   if ($multiversion_manager->isEnabledEntityType($entity_type) || $entity_type_id == 'replication_log') {
-    return AccessResult::allowedIfHasPermission($account, 'perform content replication');
+    return AccessResult::allowedIfHasPermissions($account, ['perform push replication', 'perform pull replication'], 'OR');
```

Comment:
Also, I don't think we have tests for our hook_entity_access() logic, do we? We only have access tests on a HTTP resource level, but not on entity access level.
Let's add some tests for this as well.